src/peft/tuners/tuners_utils.py

```diff
@@ -769,7 +769,9 @@ def inject_adapter(
         module_names: set[str] = set()
         if state_dict is not None:
             prefix = PEFT_TYPE_TO_PREFIX_MAPPING[peft_config.peft_type]
-            module_names = {k.rsplit("." + prefix, 1)[0] for k in state_dict}
+            # Find the module name from the state_dict. Also defensively remove '_orig_mod.', which might be inserted if
+            # the model was torch.compiled beforehand
+            module_names = {k.rsplit("." + prefix, 1)[0].replace("_orig_mod.", "") for k in state_dict}
```

Comment:
Why can we do `removeprefix` below but here we use `replace`? If possible it'd be nice to only use `removeprefix` to avoid accidental removes